src/web/server.rs

```diff
@@ -79,34 +106,21 @@ impl Handler<message::Content> for ChatServer {
 
         msg.chat_message.message = clean(&msg.chat_message.message);
 
-        // emojis = Vec<(String, String, String) where names are (find, replace, name)
-        let mut replacements: HashMap<usize, String> =
-            HashMap::with_capacity(msg.chat_message.emojis.len());
-        let mut replacement_string = msg.chat_message.message.to_owned();
-
-        // First, replace all instances with tokens.
-        for (find, replace, name) in &msg.chat_message.emojis {
-            let url = clean(replace);
-            let key: usize = rand::random();
-            let value: String = format!(
-                "<img class=\"emoji\" src=\"{}\" data-emoji=\"{}\" alt=\"{}\" />",
-                url, name, name
-            );
-            replacement_string = replacement_string.replace(find, &format!("<{}>", key));
-            replacements.insert(key, value);
-        }
+        let mut chat_msg = msg.chat_message;
 
-        // Replace tokens with real replacements.
-        for (key, value) in replacements {
-            replacement_string = replacement_string.replace(&format!("<{}>", key), &value);
+        // Check if the userscript failed to generate a UUID.
+        if chat_msg.id.is_nil() {
```

Comment:
Setting a nil `UUID` in the default `Message` and checking if it's still present here helps with debugging if it somehow doesn't get overwritten during deserialization.